.circleci/config.yml

```diff
@@ -158,6 +158,9 @@ workflows:
       - prep-build-test-mv2:
```

Comment:
Apart from the change already suggested by Mariona. This file looks good to me :+1: